Felienne/hedy
# [BUG FIX] Content container no longer shifts on scrollbar

**Description**
Currently when a scrollbar appears the container (with all content) shifts slightly to the left. This makes change as the screen gets slightly smaller and it automatically centers. However, this shifting when navigating through pages or expanding sections is annoying. In this PR we implement a fix ensuring that the content container is always in the middle of the screen, independent of a scrollbar.

**Fixes #1963**

**How to test**
Navigate to a page with not much content such as the login page. Then shift to the learn-more page and notice that the content container no longer shifts. Verify that this shifting is the case in the current implementation. See the video in the related issue for clarification.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
